RallyHereIntegration/Source/RallyHereIntegration/Private/RallyHereIntegrationModule.cpp

```diff
@@ -6,15 +6,29 @@
 #include "RallyHereAPIModule.h"
 #include "Misc/ConfigCacheIni.h"
 #include "RH_Polling.h"
+#include "Interfaces/IPluginManager.h"
 
 IMPLEMENT_MODULE(FRallyHereIntegrationModule, RallyHereIntegration);
 FString GRallyHereIntegrationIni;
 
 void FRallyHereIntegrationModule::StartupModule()
 {
 	FModuleManager::LoadModuleChecked<FRallyHereAPIModule>(FName(TEXT("RallyHereAPI")));
+	FModuleManager::LoadModuleChecked<IModuleInterface>(FName(TEXT("HTTP")));
+
+	FString PluginVersion = TEXT("Unknown");
+	auto RHIntegrationPlugin = IPluginManager::Get().FindPlugin(TEXT("RallyHereIntegration"));
+	if (RHIntegrationPlugin != nullptr)
+	{
+		const auto& Descriptor = RHIntegrationPlugin->GetDescriptor();
+		PluginVersion = Descriptor.VersionName;
+	}
+
+    UE_LOG(LogRallyHereIntegration, Log, TEXT("[%s] - Plugin Version %s"), ANSI_TO_TCHAR(__FUNCTION__), *PluginVersion);
+
+	FString UserAgentComment = FString::Printf(TEXT("rallyhere-sdk-ver=%s"), *PluginVersion);
```

Comment:
Looking at a few [examples of these comments](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#specifications), they seem to be of the form `thing:version` and try to keep it pretty short.  We might want to do something simple like `rhsdk:1.2.3`.